manifest/v1alpha/service/service.go

```diff
@@ -56,10 +56,19 @@ type ReviewCycleStatus struct {
 
 // Spec holds detailed information specific to Service.
 type Spec struct {
-	Description string       `json:"description" validate:"description" example:"Bleeding edge web app"`
+	// Description is a human-readable description of the service.
+	Description string `json:"description" validate:"description" example:"Bleeding edge web app"`
```

Comment:
I know you haven't changed that, but I noticed it (boy-scout) and these are remnants of the past and no longer used :)

```suggestion
	Description string `json:"description"`
```